core/lib/CoreProcess.ts

```diff
@@ -42,28 +42,4 @@ export default class CoreProcess {
       return closed;
     }
   }
-
-  private static getEnvironmentVariables() {
-    const processEnv: Dict<string> = {
-      SA_TEMPORARY_CORE: 'true',
-    };
-
-    // whitelist env so we don't leak sensitive args out of this process.
-    for (const [key, value] of Object.entries(process.env)) {
-      const isWhitelisted =
-        key === 'SSLKEYLOGFILE' ||
-        key === 'DEBUG' ||
-        // legacy arg
-        key === 'SHOW_BROWSER' ||
-        key.startsWith('NODE_') ||
-        key.startsWith('MITM_') ||
-        key.startsWith('SA_') ||
-        key.includes('CHROME_');
-
```

Comment:
Why get rid of the idea of whitelisting?

Reply:
1. There are bash/env variables that are not passed through to a Core instance during launch with this approach. In the case of Ubuntu, those variables seem to include things related to X windows and launching.
2. I realized that the use case of just forking off a CoreProcess on the fly where it might inherit potentially sensitive process env variables is likely to be not as much an issue when you deploy to a server and launch a Core.

Reply:
Does this expose all their ENV vars (i.e., database passwords, etc) in the session db?

Reply:
Not in the session db. We don't store all env variables anywhere.